pyoverkiz/models.py

```diff
@@ -151,8 +153,8 @@ class Device:
     controllable_name: str
     definition: Definition
     data_properties: list[dict[str, Any]] | None = None
-    widget: str | None = None
```

Comment:
Do we know why these were optional? I think every device has a widget and uiclass set.